pages/[network]/parity.tsx

```diff
@@ -1,7 +1,8 @@
 import { useContext, useEffect, useState } from "react";
 import { ListGroup, OverlayTrigger, Tooltip } from "react-bootstrap";
 
-import { NetworkFactory, toSmartContractDecimals } from "bepro-js";
+import { NetworkFactory, toSmartContractDecimals } from "@taikai/dappkit";
```

Comment:
with the `deploy` script and with the new environments this page can be removed :)